packages/mendel-pipeline/src/client/base-client.js

```diff
@@ -29,7 +29,7 @@ class BaseMendelClient extends EventEmitter {
             );
 
         this.registry = new MendelClientRegistry(this.config);
-        this.generators = new MendelGenerators(this.config, this.registry);
+        this.generators = new MendelGenerators(this.config, this.registry, options.printEntryCount);
```

Comment:
Please do not add another argument to the constructor as first argument should already have it (if anything).